### PR TITLE
ascii namelist files used by the infrared cloud and aerosol detection…

### LINKED ISSUE #18: Infrared cloud and aerosol detection software (CADS) ascii namelist files for the GSI
A new infrared cloud detection routine has been added to the GSI.  Specific quality control / cloud detection files are needed for each instrument using this new routine.  At this time, the GSI is setup to use CADS for the AIRS, CrIS and IASI instruments.  These files contain the quality control / cloud detection parameters for these 3 instruments.

These 3 files reside in https://github.com/wx20jjung/GSI-fix.git in the fix-CADS branch.

### PR DESCRIPTION
**Pull request by wx20jjung**
… software (CADS) added to the GSI.  These files are specific to the AIRS, CrIS and IASI instruments.

Resolves #18

### CODE REVIEW

**Comment by RussTreadon-NOAA** [comment]
@wx20jjung , I added myself as a reviewer.  We need one more reviewer.  Who would you like me to add as the other reviewer?

File: AIRS_CLDDET.NL [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+                   83,  84,  86,  92,  93,  98,  99, 101, 104, 105,
+                  108, 110, 111, 113, 116, 117, 123, 124, 128, 129,
+                  138, 139, 144, 145, 150, 151, 156, 157, 159, 162,
+                  165, 168, 169, 170, 172, 173, 174, 175, 177, 179,
+                  180, 182, 185, 186, 190, 192, 193, 198, 201, 204,
+                  207, 210, 213, 215, 216, 218, 221, 224, 226, 227,
+                  232, 239, 248, 250, 251, 252, 253, 256, 257, 261,
+                  262, 267, 272, 295, 299, 305, 308, 309, 310,
+                  318, 321, 333, 338, 355, 362, 375, 475,
+                  484, 497, 528, 587, 672, 787, 791, 843, 870, 914,
```
Comment (DavidHuber-NOAA) [inline AIRS_CLDDET.NL]: Minor nitpick: is there are reason for not extending these lines out to 10 bands each?

File: CRIS_CLDDET.NL [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+N__Bands(1:12,4)=
+ 1149, 1157, 1164, 1165, 1173, 1181, 1189, 1197, 1205, 1213,
+ 1221, 1251
+R__BT_Threshold(1)=0.60
+N__Window_Width(1)=6
+L__Do_Imager_Cloud_Detection = .TRUE.
+N__Num_Imager_Chans = 2
+N__Num_Imager_Clusters = 7
+N__Imager_Chans(:) = 15, 16
+R__Stddev_Threshold(:) = 0.4, 0.4
```
Comment (DavidHuber-NOAA) [inline CRIS_CLDDET.NL]: All of your other namelist arrays are explicitly indexed.  Should these be as well?  i.e.:
```
N__Imager_Chans(1:2) = 15, 16
R__Stddev_Threshold(1:2) = 0.4, 0.4
```

File: IASI_CLDDET.NL [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+           515,  546,  552,  566,  571,  573,  646,  662,  668,  756,  
+           867,  921, 1027, 1090, 1133, 1191, 1194, 1271, 1805, 1884,
+          1946, 1991, 2094, 2239
+N__GradChkInterval(1:5) = 12,5,5,5,5
+R__BT_Threshold(1)=0.6
+L__Do_Imager_Cloud_Detection = .TRUE.
+N__Num_Imager_Chans = 2
+N__Num_Imager_Clusters = 7
+N__Imager_Chans(:) = 2, 3 
+R__Stddev_Threshold(:) = 0.60, 0.60
```
Comment (DavidHuber-NOAA) [inline IASI_CLDDET.NL]: Should these also be explicitly indexed?

**Review by DavidHuber-NOAA — COMMENTED** [review]
I can't comment on the technicals behind these channel selections, but have a few questions about the way things were coded.

**Comment by wx20jjung** [comment]
Dave,

In the AIRS_CLDDET.NL
I believe the gaps are channels that failed over the past 20+ years.  ECMWF
just removed the channels and did not adjust the blocks.  I will change
them.

The other two, I will add the specific array bounds.

I will need to run the analyses again to make sure nothing is broken.  It
will probably be this PM before I have these completed and resubmitted.
Jim

**Comment by wx20jjung** [comment]
Changes are pushed back to github.  Results of changes are identical.

**Review by DavidHuber-NOAA — APPROVED** [review]
Looks good!  Thanks Jim!

**Review by RussTreadon-NOAA — APPROVED** [review]
Thank you @DavidHuber-NOAA for your review.  Thank you @wx20jjung for making & testing requested changes.

Approve.

**Comment by wx20jjung** [comment]
Thanks Russ and Dave!  I will start the process to move these changes into
the global workflow.

Jim

**Pull request merged**